.github/workflows/test_python.yaml

```diff
@@ -63,33 +63,20 @@ jobs:
       - name: Install Python Tools
         run: |
           pip install -U pip tox codecov coverage
-      # Sphinx 3.0.0 ###################################################################
-      - name: Test Python ${{ matrix.python-version }} / sphinx==3.0.0
+      # Sphinx 4.0.0 ###################################################################
+      - name: Test Python ${{ matrix.python-version }} / sphinx==4.0.0
+        # Sphinx 4 appears to by python < 3.10.
+        if: ${{ matrix.python-version == '3.8' || matrix.python-version == '3.9' }}
         env:
           # NOTE: this is the current minimum supported.
-          SPHINX_VERSION: '==3.0.0'
+          SPHINX_VERSION: '==4.0.0'
```

Comment:
`4.0.0` is the first release. Better chose latest patch level `4.0.3`.

Reply:
Agreed ... trying `4.3.2` because I'm pretty sure that's the oldest we actually need to support here.  Lets see what CI does ...

Reply:
I am happy with it.